src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/EventNotificationEmailManager.java

```diff
@@ -0,0 +1,170 @@
+import uk.ac.cam.cl.dtg.isaac.dto.IsaacEventPageDTO;
+import uk.ac.cam.cl.dtg.isaac.dto.ResultsWrapper;
+import uk.ac.cam.cl.dtg.isaac.dto.content.ContentDTO;
+import uk.ac.cam.cl.dtg.isaac.dto.eventbookings.DetailedEventBookingDTO;
+import uk.ac.cam.cl.dtg.isaac.dto.users.RegisteredUserDTO;
+import uk.ac.cam.cl.dtg.isaac.dto.users.UserSummaryDTO;
+import uk.ac.cam.cl.dtg.segue.api.Constants;
+import uk.ac.cam.cl.dtg.segue.api.managers.UserAccountManager;
+import uk.ac.cam.cl.dtg.segue.api.services.ContentService;
+import uk.ac.cam.cl.dtg.segue.auth.exceptions.NoUserException;
+import uk.ac.cam.cl.dtg.segue.comm.EmailManager;
+import uk.ac.cam.cl.dtg.segue.comm.EmailType;
+import uk.ac.cam.cl.dtg.segue.dao.SegueDatabaseException;
+import uk.ac.cam.cl.dtg.segue.dao.content.ContentManagerException;
+import uk.ac.cam.cl.dtg.segue.dao.content.IContentManager;
+import uk.ac.cam.cl.dtg.segue.search.AbstractFilterInstruction;
+import uk.ac.cam.cl.dtg.segue.search.DateRangeFilterInstruction;
+import uk.ac.cam.cl.dtg.util.PropertiesLoader;
+
+import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static uk.ac.cam.cl.dtg.isaac.api.Constants.*;
+import static uk.ac.cam.cl.dtg.isaac.api.Constants.DATE_FIELDNAME;
+import static uk.ac.cam.cl.dtg.segue.api.Constants.*;
+
+public class EventNotificationEmailManager {
+    private static final Logger log = LoggerFactory.getLogger(EventNotificationEmailManager.class);
+
+    private final PropertiesLoader properties;
+    private final IContentManager contentManager;
+    private final EventBookingManager bookingManager;
+    private final UserAccountManager userAccountManager;
+    private final EmailManager emailManager;
+    private final PgScheduledEmailManager pgScheduledEmailManager;
+
+
+    /**
+     * This class is required by quartz and must be executable by any instance of the segue api relying only on the
+     * jobdata context provided.
+     */
+    @Inject
+    public EventNotificationEmailManager(final PropertiesLoader properties,
+                                         final IContentManager contentManager,
+                                         final EventBookingManager bookingManager,
+                                         final UserAccountManager userAccountManager,
+                                         final EmailManager emailManager,
+                                         final PgScheduledEmailManager pgScheduledEmailManager) {
+        this.properties = properties;
+        this.contentManager = contentManager;
+        this.bookingManager = bookingManager;
+        this.userAccountManager = userAccountManager;
+        this.emailManager = emailManager;
+        this.pgScheduledEmailManager = pgScheduledEmailManager;
+    }
+
+    public void sendEmailForEvent(IsaacEventPageDTO event, String templateId) throws SegueDatabaseException {
```

Comment:
I got confused by this method name, because it's singular and unclear who is being emailed. If (see below) we filter for all confirmed/reserved/non-cancelled bookings then the title should imply that?
`sendEmailForConfirmedBookings(...)`, `sendEmailToAttendees(...)` ?